backend/api/views.py

```
@@ -1,75 +1,163 @@
-            "user": {
-                "id": request.user.id,
-                "username": request.user.username,
-                "email": request.user.email
-            }
+            
         })
 
-class OnboardingAPIView(APIView):
+class OnboardingAPIView(generics.UpdateAPIView):
+
     permission_classes = [IsAuthenticated]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)